owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListComposeFragment.kt

```diff
@@ -0,0 +1,532 @@
+                fontWeight = FontWeight.Bold,
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                fontSize = 12.sp,
+                text = spaceName
+            )
+            Text(
+                modifier = Modifier
+                    .layoutId("path"),
+                color = colorResource(id = R.color.list_item_lastmod_and_filesize_text),
+                overflow = TextOverflow.Ellipsis,
+                maxLines = 1,
+                fontSize = 12.sp,
+                text = parentPath
+            )
+        }
+    }
+
+    @Composable
+    fun EmptyList() {
+        Column(
+            modifier = Modifier
+                .padding(top = 8.dp, bottom = 8.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center
+        ) {
+            Icon(
+                modifier = Modifier
+                    .layoutId("list_empty_dataset_icon")
+                    .width(72.dp)
+                    .height(72.dp),
+                tint = colorResource(id = R.color.grey),
+                painter = painterResource(id = R.drawable.ic_uploads),
+                contentDescription = "Empty dataset Icon"
+            )
+            Text(
+                modifier = Modifier
+                    .padding(top = 8.dp, bottom = 8.dp, start = 16.dp, end = 16.dp),
+                color = colorResource(id = R.color.half_black),
+                fontFamily = FontFamily.SansSerif,
+                fontStyle = FontStyle.Normal,
+                fontSize = 16.sp,
+                letterSpacing = 0.009375.sp,
+                fontWeight = FontWeight.Bold,
+                text = stringResource(id = R.string.upload_list_empty)
+            )
+            Text(
+                modifier = Modifier
+                    .layoutId("list_empty_dataset_sub_title")
+                    .padding(start = 8.dp, end = 8.dp),
+                color = colorResource(id = R.color.grey),
+                fontFamily = FontFamily.SansSerif,
+                fontStyle = FontStyle.Normal,
+                fontSize = 16.sp,
+                letterSpacing = 0.03125.sp,
+                text = stringResource(id = R.string.upload_list_empty_subtitle)
+            )
+        }
+
+    }
+
+    @Composable
+    fun UploadGroup(status: TransferStatus, count: Int) {
+        val stringResFileCount =
+            if (count == 1) R.string.uploads_view_group_file_count_single else R.string.uploads_view_group_file_count
+        val fileCountText: String = String.format(requireContext().getString(stringResFileCount), count)
+
+        Column {
+            Row(
+                modifier = Modifier
+                    .padding(top = 10.dp, bottom = 5.dp)
+                    .fillMaxWidth()
+            ) {
+                Text(
+                    modifier = Modifier
+                        .layoutId("uploadListGroupName")
```

Comment:
Same with this layout ID and all the others

Reply:
Done ✅